### PR TITLE
slapt-get: add page

### PR DESCRIPTION
**Pull request by alinepegas**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/slapt-get.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# slapt-get
+
+> An apt like system for Slackware package management.
+> Package sources need to be configured on the slapt-getrc file.
```
Comment (sbrl) [inline pages/linux/slapt-get.md]: `on` -> `in`?

File: pages/linux/slapt-get.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Update the list of available packages and versions (it's recommended to run this before other `slapt-get` commands):
```
Comment (sbrl) [inline pages/linux/slapt-get.md]: Not sure that the `it's recommended` bit in brackets is needed. Generally, we order things in a tldr-page in the order that one would execute them. Additionally, it's rather similar to the `apt update` paradigm that's used in multiple other package managers.

What do you think?

File: pages/linux/slapt-get.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`slapt-get --update`
+
+- Install a package, or update it to the latest available version:
+
+`slapt-get --install {{package}}`
```
Comment (sbrl) [inline pages/linux/slapt-get.md]: Perhaps `package_name` would be more descriptive?

File: pages/linux/slapt-get.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Remove a package:
+
+`slapt-get --remove {{package}}`
+
+- Upgrade all installed packages to their newest available versions:
```
Comment (sbrl) [inline pages/linux/slapt-get.md]: Perhaps ...`to their latest versions` would be more consistent?

File: pages/linux/slapt-get.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`slapt-get --upgrade {{package}}`
+
+- Locate packages of interest by the package name, disk set, or version:
```
Comment (sbrl) [inline pages/linux/slapt-get.md]: What's a _disk set_? Not a _Slackware_ user, so I'm unsure :P

File: pages/linux/slapt-get.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`slapt-get --search {{package}}`
+
+- Show information about packages:
```
Comment (sbrl) [inline pages/linux/slapt-get.md]: Perhaps ...`about a package` would read better?

File: pages/linux/slapt-get.md (hunk 1 above)
Comment (alinepegas) [reply pages/linux/slapt-get.md]: Sure

File: pages/linux/slapt-get.md (hunk 2 above)
Comment (alinepegas) [reply pages/linux/slapt-get.md]: Well, I agree that it's unnecessary. I put it there to mimic what was done on the `apt-get.md` page. Should I keep it or remove it?

File: pages/linux/slapt-get.md (hunk 3 above)
Comment (alinepegas) [reply pages/linux/slapt-get.md]: Yes, I'll make that change.

File: pages/linux/slapt-get.md (hunk 4 above)
Comment (alinepegas) [reply pages/linux/slapt-get.md]: I agree, I will make the change.

File: pages/linux/slapt-get.md (hunk 5 above)
Comment (alinepegas) [reply pages/linux/slapt-get.md]: Slackware used to be distribuited as a set of iso images (multiple CDs) where each package category (packages for gnome, packages for kde, packages for developers)  were in a different disk. Now there is only one ISO, but the package categories remain and are still called disk sets.

File: pages/linux/slapt-get.md (hunk 6 above)
Comment (alinepegas) [reply pages/linux/slapt-get.md]: Yes, I agree. I will make the change.

File: pages/linux/slapt-get.md (hunk 5 above)
Comment (sbrl) [reply pages/linux/slapt-get.md]: Ah, I see :-)

**Pull request merged**